barretenberg/sol/src/ultra/keys/RecursiveUltraVerificationKey.sol

```diff
@@ -14,52 +14,52 @@ library RecursiveUltraVerificationKey {
             mstore(add(_vk, 0x20), 0x0000000000000000000000000000000000000000000000000000000000000010) // vk.num_inputs
             mstore(add(_vk, 0x40), 0x2260e724844bca5251829353968e4915305258418357473a5c1d597f613f6cbd) // vk.work_root
             mstore(add(_vk, 0x60), 0x3064486657634403844b0eac78ca882cfd284341fcb0615a15cfcd17b14d8201) // vk.domain_inverse
```

Comment:
These verifications keys were stale because the unit tests were failing for the UltraPlonk contract